### PR TITLE
Adding flux config for S2S in AAT.

### PR DESCRIPTION
**Pull request by adusumillipraveen**
Adding flux config for S2S in AAT.

### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RPE-984

### Change description ###

Added flux config for S2S chart in AAT. 

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: k8s/aat/software-engineering/rpe-service-auth-provider/rpe-service-auth-provider.yaml [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+  releaseName: rpe-service-auth-provider
+  chart:
+    repository: https://hmcts.azurecr.io/helm/v1/repo
+    name: rpe-service-auth-provider
+    version: 0.0.1
+  values:
+    java:
+      applicationPort: 8489
+      keyVaults:
+        "s2s":
```
Comment (timja) [inline k8s/aat/software-engineering/rpe-service-auth-provider/rpe-service-auth-provider.yaml]: isn't this already in the chart?

File: k8s/aat/software-engineering/rpe-service-auth-provider/rpe-service-auth-provider.yaml [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+            - microservicekey-cet-frontend
+            - microservicekey-fpl-case-service
+            - microservicekey-iac
+            - microservicekey-em-stitching-api
+            - microservicekey-em-ccd-orchestrator
+            - microservicekey-ccpay-bubble
+            - microservicekey-dg-template-management
+            - microservicekey-dg-docassembly-api
+      APPINSIGHTS_INSTRUMENTATIONKEY: "ed3c45ae-01e1-4263-b57f-178a1ba004be"
+      image: hmcts/service-auth-provider-app
```
Comment (timja) [inline k8s/aat/software-engineering/rpe-service-auth-provider/rpe-service-auth-provider.yaml]: we want to pull the version that's published in ACR, see the cmc-pdf-service one

File: k8s/aat/software-engineering/rpe-service-auth-provider/rpe-service-auth-provider.yaml (hunk 1 above)
Comment (adusumillipraveen) [reply k8s/aat/software-engineering/rpe-service-auth-provider/rpe-service-auth-provider.yaml]: They are in values.template.yaml . I am assuming they won't be picked?

Comment (timja) [reply k8s/aat/software-engineering/rpe-service-auth-provider/rpe-service-auth-provider.yaml]: they will be

**Pull request merged**